### ISSUE TITLE
Add Electra Weights to Kaggle Models

### ISSUE THREAD

**Issue by pranavvp16** [keras-team/keras-hub#1422]
Is there any certain guide to upload weights of pretrained models to kaggle. As per the `CONTRIBUTING_MODELS.md` I have uploaded my weights on GCP as mentioned in #1384.

**Comment by abuelnasr0** [comment]
Hi @pranavvp16 
You can follow the @mattdangerw  guide in this comment https://github.com/keras-team/keras-nlp/pull/1382#issuecomment-1891286724

**Comment by SuryanarayanaY** [comment]
Hi @pranavvp16 ,

Could you please confirm whether this has been resolved now? Please feel free to close the issue if reoslved. Thanks!

**Comment by pranavvp16** [comment]
Yes this one is resolved now, Also it would be nice if we update the `CONTRIBUTING_MODELS.md` with the latest steps to contribute a pretrained model in keras_nlp. Let me know I can update it with the steps to upload the model on kaggle rather than GCP or google drive.

**Comment by dhantule** [comment]
Hi @pranavvp16, feel free close this issue if it's resolved.

> Yes this one is resolved now, Also it would be nice if we update the `CONTRIBUTING_MODELS.md` with the latest steps to contribute a pretrained model in keras_nlp. Let me know I can update it with the steps to upload the model on kaggle rather than GCP or google drive.

please raise another issue, describing the contribution you want to make.
